fastpath/fastpath/core.py

```diff
@@ -914,27 +914,24 @@ def score_vanilla_tor(msm: dict) -> dict:
     tk = g_or(msm, "test_keys", {})
     scores = init_scores()
 
-    nks = ("error", "success", "tor_log", "tor_progress_summary", "tor_progress_tag")
+    nks = ("error", "success", "tor_logs", "tor_progress_summary", "tor_progress_tag")
     if msm["software_name"] == "ooniprobe" and all_keys_none(tk, nks):
```

Comment:
I think that since you are looking specifically for the `software_name` `ooniprobe` (i.e. the python implementation), you should be using the `tor_log` key instead of `tor_logs`, since that's the value used in legacy OONI Probe: https://github.com/ooni/spec/blob/master/nettests/ts-016-vanilla-tor.md#incompatibility-with-ooniprobe-legacy

Reply:
Updated as discussed.